testutils/file.go

```diff
@@ -0,0 +1,12 @@
+package testutils
+
+import (
+	"os"
+	"path"
+)
+
+func TestDataFile(name string) string {
+	dir, _ := os.Getwd()
+
+	return path.Join(dir, "testdata", name)
```

Comment:
Would it make sense to create an empty file here, perhaps in a tmp dir, instead of checking empty files into Git?

Reply:
We could do that but I don't necessarily think empty files in git are a problem for testing, and makes this slightly more complicated with testing (really stressing the slightly in that part). I, of course, don't have a problem if you'd prefer me creating files in the tmp directory

Reply:
OK, no worries.  I don't have a strong feeling here, and when that's the case I think it's better to defer to what you've already implemented.